lib/run/index.js

```diff
@@ -112,7 +112,7 @@ module.exports = function (options, callback) {
             },
             certificates: options.sslClientCert && new sdk.CertificateList({}, [{
                 name: 'client-cert',
-                matches: ['<all_urls>'],
+                matches: ['<all_urls>'], // todo - expose this as a constant from the SDK.
```

Comment:
My bad 😞 

`UrlMatchPattern` is [already exposed](https://github.com/postmanlabs/postman-collection/blob/develop/lib/index.js#L23) from `postman-collection`.

You can use [`require('postman-collection').UrlMatchPattern.MATCH_ALL_URLS`](https://github.com/postmanlabs/postman-collection/blob/develop/lib/url-pattern/url-match-pattern.js#L264)